src/main/java/com/amazon/opendistroforelasticsearch/security/support/Base64Helper.java

```diff
@@ -38,79 +44,123 @@
 import java.io.ObjectInputStream;
 import java.io.ObjectOutputStream;
 import java.io.ObjectStreamClass;
+import java.io.OutputStream;
 import java.io.Serializable;
 import java.net.InetAddress;
 import java.net.InetSocketAddress;
 import java.net.SocketAddress;
-import java.util.ArrayList;
+import java.security.PrivilegedAction;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
+import java.util.regex.Pattern;
 
 import org.elasticsearch.ElasticsearchException;
+import org.elasticsearch.common.Strings;
 
-import com.amazon.opendistroforelasticsearch.security.resolver.IndexResolverReplacer;
```

Comment:
Is `IndexResolverReplacer.Resolved` not needed or missed ?

Reply:
It is not needed.